AUTHORS

```diff
@@ -7,3 +7,4 @@
 # The email address is not required for organizations.
 
 Google Inc.
+Nathan Naze <nanaze@gmail.com>
```

Comment:
If you are currently working at Google, remove your name, Google owns the copyright.